coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -106,6 +110,18 @@ def markers(self):
         """
         return self._markers
 
+    @property
+    def metadata(self):
+        """
+        A namedtuple of documentation symbols.
+
+        These documentation symbols are used to define certain symbols that
+        define a documentation comment.
```

Comment:
hm they don't really define the comment, they contain more information about special attributes inside the documentation (I think). Imo you should rephrase :+1: